packages/tables/src/Columns/Concerns/HasRecord.php

```diff
@@ -19,4 +19,9 @@ public function getRecord(): ?Model
     {
         return $this->record ?? $this->getLayout()?->getRecord();
     }
+
+    public function hasRecord(): bool
+    {
+        return $this->record !== null;
+    }
```

Comment:
Please move this to the `Filament\Actions\Contracts\HasRecord` interface, it just needs the method signature not the implementation